modules/volunteers/admin/sitesupport/vue.js

```diff
@@ -61,6 +66,50 @@ var app = Vue.createApp({
           const data = JSON.parse(response.responseText);
           this.reward_groups = data.data;
         })
+    },
+    clearMember() {
+      this.userId = null;
+      this.userData = null;
+      this.$refs.vhtbl.load();
+      this.$refs.gfttbl.load();
+      this.$refs.lookup.clear();
+    },
+    enterAdmin() {
+      var target = '';
+      if (this.userId) {
+        target = 'index.php?Function=volunteers/admin&volunteerId=' + this.userId;
+      } else {
+        target = 'index.php?Function=volunteers/admin';
+      }
+      setTimeout(function() {window.location = target;}, 1000);
+    },
+    failAdmin(error) {
+      document.getElementById('admin_slider').checked = false;
+      if (error) {
+        alertbox('Login Failed (' + error + ')');
+      }
+    },
+    toggleAdminMode() {
+      document.cookie = 'CIAB_VOLUNTEERADMIN=;expires=Thu, 01 Jan 1970 ' +
+                        '00:00:01 GMT;';
+      var target = '';
```

Comment:
Can we use `let` here instead of `var`?

Additionally this could be a good use case for `URLSearchParams`:

```
const searchParams = new URLSearchParams('Function=volunteers/admin');

if (this.userId) {
  searchParams.append(`volunteerId=${this.userId}`);
}

const target = `index.php?${searchParams.toString()}`;
```

Reply:
Same `%2f` thing as above